mypy/fscache.py

```diff
@@ -54,6 +54,7 @@ def stat(self, path: str) -> os.stat_result:
         try:
             st = os.stat(path)
         except Exception as err:
+            err.__traceback__ = None
```

Comment:
This didn't actually reclaim the space used by the traceback in my experiments. I tried something different and was able to save some memory use. I'll create a PR once I've verified that my approach works.

Reply:
Even after a `gc.collect()`?